### ISSUE TITLE
Error: template not found

### ISSUE THREAD

**Issue by djheru** [infinitered/gluegun#439]
I've just installed gluegun using `yarn global add gluegun` which installed version 2.0.5. Then I attempted to follow along with the "Making a movie CLI" tutorial by running `gluegun new movie --typescript`, which threw the following error:

```
 ~/Workspace/_sandbox  gluegun new movie --typescript
/Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: template not found /Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/cli/templates/cli/.gitignore.ejs
    at /Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/toolbox/template-tools.js:71:27
    at step (/Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/toolbox/template-tools.js:32:23)
    at Object.next (/Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/toolbox/template-tools.js:13:53)
    at /Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/toolbox/template-tools.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/toolbox/template-tools.js:3:12)
    at generate (/Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/toolbox/template-tools.js:51:16)
    at /Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/cli/commands/new.js:99:35
    at Array.reduce (<anonymous>)
    at Object.<anonymous> (/Users/philip/.nvm/versions/node/v8.12.0/lib/node_modules/gluegun/build/cli/commands/new.js:95:36)
```

This is on mac running node 8.12

**Comment by djheru** [comment]
As an FYI, I uninstalled and re-installed `gluegun@next`. It installed 2.0.0, and I did not receive the same error when generating a new cli project

**Comment by jamonholmgren** [comment]
I've replicated the issue; working on a fix. Not sure how this got through CI!

**Comment by jamonholmgren** [comment]
Fixed in 2.0.7 (2.0.6 was an aborted attempt).

https://github.com/infinitered/gluegun/releases/tag/v2.0.7

Thanks for reporting @djheru !
